lib/widgets/recent_dm_conversations.dart

```diff
@@ -101,8 +133,26 @@ class RecentDmConversationsItem extends StatelessWidget {
               maxLines: 2,
               overflow: TextOverflow.ellipsis,
               title))),
-          const SizedBox(width: 8),
-          // TODO(#253): Unread count
+          unreadCount > 0
+            ? Container(
+                margin: const EdgeInsetsDirectional.only(start: 8, end: 16),
```

Comment:
```suggestion
            ? Padding(
                padding: const EdgeInsetsDirectional.only(start: 8, end: 16),
```

That's what `Container.margin` boils down to — see the implementation of `Container.build`. And I find it a lot clearer to understand this way, particularly because it avoids sounding like CSS's "margin", which is a lot more complicated.